packages/neuron-wallet/.env

```diff
@@ -1,15 +1,23 @@
+# mainnet
 MAINNET_SUDT_DEP_TXHASH=0xc7813f6a415144643970c2e88e0bb6ca6a8edc5dd7c1022746f628284a9936d5
 MAINNET_SUDT_DEP_INDEX=0
 MAINNET_SUDT_DEP_TYPE=code
 MAINNET_SUDT_SCRIPT_CODEHASH=0x5e7a36a77e68eecc013dfa2fe6a23f3b6c344b04005808694ae6dd45eea4cfd5
 MAINNET_SUDT_SCRIPT_HASHTYPE=type
-MAINNET_ACP_DEP_TXHASH=0xa05f28c9b867f8c5682039c10d8e864cf661685252aa74a008d255c33813bb81
+
+MAINNET_ACP_DEP_TXHASH=0x4153a2014952d7cac45f285ce9a7c5c0c0e1b21f2d378b82ac1433cb11c25c4d
 MAINNET_ACP_DEP_INDEX=0
 MAINNET_ACP_DEP_TYPE=depGroup
-MAINNET_ACP_SCRIPT_CODEHASH=0x0fb343953ee78c9986b091defb6252154e0bb51044fd2879fde5b27314506111
+MAINNET_ACP_SCRIPT_CODEHASH=0xd369597ff47f29fbc0d47d2e3775370d1250b85140c670e4718af712983a2354
 MAINNET_ACP_SCRIPT_HASHTYPE=data
```

Comment:
acp script hashtype should be `type`

Reply:
Really nice catch. Thanks a lot @shaojunda